### ISSUE TITLE
Loss function for supervised loss 

### ISSUE THREAD

**Issue by sb1992** [GT-SALT/MixText#14]
I have a question about what exactly is the supervised loss mentioned in paper or here in the code this Lx. It is not exactly cross entropy so was a bit confused, a bit more explanation about this loss function would be really helpful?
 
```Lx = - torch.mean(torch.sum(F.log_softmax(outputs_x, dim=1) * targets_x, dim=1))```

**Comment by jiaaoc** [comment]
Here it is actually still cross-entropy (except that the targets_x is not a one-hot label). If your targets_x is a one-hot label, you could view it as the general CE loss.

**Comment by sb1992** [comment]
ah yes, correct. thank you for clearing it up.
